from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_all_in_one(object):
    def setupUi(self, all_in_one):
        all_in_one.setObjectName("all_in_one")
        all_in_one.resize(788, 260)
        self.tabWidget = QtWidgets.QTabWidget(all_in_one)
        self.tabWidget.setGeometry(QtCore.QRect(0, 0, 791, 261))
        self.tabWidget.setObjectName("tabWidget")
        self.tab = QtWidgets.QWidget()
        self.tab.setObjectName("tab")
        self.label = QtWidgets.QLabel(self.tab)
        self.label.setGeometry(QtCore.QRect(10, 10, 761, 91))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(12)
        self.label.setFont(font)
        self.label.setAlignment(QtCore.Qt.AlignCenter)
        self.label.setObjectName("label")
        self.pushButton = QtWidgets.QPushButton(self.tab)
        self.pushButton.setGeometry(QtCore.QRect(290, 160, 201, 41))
        self.pushButton.setObjectName("pushButton")
        self.tabWidget.addTab(self.tab, "")
        self.tab_2 = QtWidgets.QWidget()
        self.tab_2.setObjectName("tab_2")
        self.pushButton_2 = QtWidgets.QPushButton(self.tab_2)
        self.pushButton_2.setGeometry(QtCore.QRect(290, 160, 201, 41))
        self.pushButton_2.setObjectName("pushButton_2")
        self.label_2 = QtWidgets.QLabel(self.tab_2)
        self.label_2.setGeometry(QtCore.QRect(10, 10, 761, 91))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(12)
        self.label_2.setFont(font)
        self.label_2.setScaledContents(False)
        self.label_2.setAlignment(QtCore.Qt.AlignCenter)
        self.label_2.setObjectName("label_2")
        self.tabWidget.addTab(self.tab_2, "")
        self.tab_3 = QtWidgets.QWidget()
        self.tab_3.setObjectName("tab_3")
        self.pushButton_3 = QtWidgets.QPushButton(self.tab_3)
        self.pushButton_3.setGeometry(QtCore.QRect(290, 160, 201, 41))
        self.pushButton_3.setObjectName("pushButton_3")
        self.label_3 = QtWidgets.QLabel(self.tab_3)
        self.label_3.setGeometry(QtCore.QRect(10, 10, 761, 91))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(12)
        self.label_3.setFont(font)
        self.label_3.setAlignment(QtCore.Qt.AlignCenter)
        self.label_3.setObjectName("label_3")
        self.tabWidget.addTab(self.tab_3, "")
        self.tab_4 = QtWidgets.QWidget()
        self.tab_4.setObjectName("tab_4")
        self.pushButton_4 = QtWidgets.QPushButton(self.tab_4)
        self.pushButton_4.setGeometry(QtCore.QRect(290, 160, 201, 41))
        self.pushButton_4.setObjectName("pushButton_4")
        self.label_4 = QtWidgets.QLabel(self.tab_4)
        self.label_4.setGeometry(QtCore.QRect(10, 10, 761, 91))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(12)
        self.label_4.setFont(font)
        self.label_4.setAlignment(QtCore.Qt.AlignCenter)
        self.label_4.setObjectName("label_4")
        self.tabWidget.addTab(self.tab_4, "")
        self.tab_5 = QtWidgets.QWidget()
        self.tab_5.setObjectName("tab_5")
        self.label_9 = QtWidgets.QLabel(self.tab_5)
        self.label_9.setGeometry(QtCore.QRect(10, 10, 761, 91))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(12)
        self.label_9.setFont(font)
        self.label_9.setAlignment(QtCore.Qt.AlignCenter)
        self.label_9.setObjectName("label_9")
        self.pushButton_9 = QtWidgets.QPushButton(self.tab_5)
        self.pushButton_9.setGeometry(QtCore.QRect(290, 160, 201, 41))
        self.pushButton_9.setObjectName("pushButton_9")
        self.tabWidget.addTab(self.tab_5, "")
        self.tab_6 = QtWidgets.QWidget()
        self.tab_6.setObjectName("tab_6")
        self.label_10 = QtWidgets.QLabel(self.tab_6)
        self.label_10.setGeometry(QtCore.QRect(10, 10, 761, 91))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(12)
        self.label_10.setFont(font)
        self.label_10.setAlignment(QtCore.Qt.AlignCenter)
        self.label_10.setObjectName("label_10")
        self.pushButton_10 = QtWidgets.QPushButton(self.tab_6)
        self.pushButton_10.setGeometry(QtCore.QRect(290, 160, 201, 41))
        self.pushButton_10.setObjectName("pushButton_10")
        self.tabWidget.addTab(self.tab_6, "")
        self.tab_7 = QtWidgets.QWidget()
        self.tab_7.setObjectName("tab_7")
        self.label_11 = QtWidgets.QLabel(self.tab_7)
        self.label_11.setGeometry(QtCore.QRect(10, 10, 761, 91))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(12)
        self.label_11.setFont(font)
        self.label_11.setAlignment(QtCore.Qt.AlignCenter)
        self.label_11.setObjectName("label_11")
        self.pushButton_11 = QtWidgets.QPushButton(self.tab_7)
        self.pushButton_11.setGeometry(QtCore.QRect(290, 160, 201, 41))
        self.pushButton_11.setObjectName("pushButton_11")
        self.tabWidget.addTab(self.tab_7, "")

        self.retranslateUi(all_in_one)
        self.tabWidget.setCurrentIndex(0) #Порядок открытия qtabwidget
        QtCore.QMetaObject.connectSlotsByName(all_in_one)

    def retranslateUi(self, all_in_one):
        _translate = QtCore.QCoreApplication.translate
        all_in_one.setWindowTitle(_translate("all_in_one", "Geoscience and coordinates"))
        self.label.setText(_translate("all_in_one", "<html><head/><body><p><span style=\" font-weight:600;\">Make empty table</span></p><p>Program that creates a table for coordinates with your number of rows.</p></body></html>"))
        self.pushButton.setText(_translate("all_in_one", "Make empty table"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab), _translate("all_in_one", "Empty table"))
        self.pushButton_2.setText(_translate("all_in_one", "Txt to xlsx"))
        self.label_2.setText(_translate("all_in_one", "<html><head/><body><p><span style=\" font-weight:600;\">Txt to xlsx</span></p><p>Program that helps to transfer data from Global Mapper export file</p><p>or your own file to a xlsx table using the right number of decimal.</p></body></html>"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab_2), _translate("all_in_one", "Txt to Xlsx"))
        self.pushButton_3.setText(_translate("all_in_one", "Txt to xlsx XY"))
        self.label_3.setText(_translate("all_in_one", "<html><head/><body><p><span style=\" font-weight:600;\">Txt to xlsx XY</span></p><p>Program that helps to transfer data from Global Mapper export file </p><p>or your own file to a xlsx table use Gauss Krueger XY coordinates.</p></body></html>"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab_3), _translate("all_in_one", "Txt to xlsx XY"))
        self.pushButton_4.setText(_translate("all_in_one", "Area calculate"))
        self.label_4.setText(_translate("all_in_one", "<html><head/><body><p><span style=\" font-weight:600;\">Area calculate</span></p><p>Program calculate area with Shoelace formula (also known as Gauss\'s area formula)</p><p>use Global Mapper export file or your own file.</p></body></html>"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab_4), _translate("all_in_one", "Area calculate"))
        self.label_9.setText(_translate("all_in_one", "<html><head/><body><p><span style=\" font-weight:600;\">Reverse export</span></p><p>Program that helps to convert data from xlsx table to</p><p>Global Mapper and import coordinates.</p></body></html>"))
        self.pushButton_9.setText(_translate("all_in_one", "Reverse export"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab_5), _translate("all_in_one", "Reverse export"))
        self.label_10.setText(_translate("all_in_one", "<html><head/><body><p><span style=\" font-weight:600;\">Reverse export XY</span></p><p>Program that helps to convert data from xlsx table to Global Mapper</p><p>and import coordinates use Gauss Krueger XY coordinates.</p></body></html>"))
        self.pushButton_10.setText(_translate("all_in_one", "Reverse export XY"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab_6), _translate("all_in_one", "Reverse export XY"))
        self.label_11.setText(_translate("all_in_one", "<html><head/><body><p><span style=\" font-weight:600;\">GeoCalc extension</span></p><p>Program that helps to convert our .xlsx table to .txt file </p><p>and back again then use it with Phgeocalc.</p></body></html>"))
        self.pushButton_11.setText(_translate("all_in_one", "GeoCalc extension"))
        self.tabWidget.setTabText(self.tabWidget.indexOf(self.tab_7), _translate("all_in_one", "GeoCalc extension"))
